flake-modules/dev/devshell.nix

```diff
@@ -81,10 +81,9 @@
 
                 echo -e "\n=> Documentation successfully built ('$doc_derivation')"
 
-                port=8000
-                echo -e "\n=> Now open your browser and navigate to 'localhost:$port'\n"
+                echo -e "\n=> You can then open your browser to view the doc\n"
 
-                ${pkgs.lib.getExe pkgs.python3} -m http.server -d "$doc_derivation"/share/doc
+                (cd "$doc_derivation"/share/doc && ${pkgs.lib.getExe pkgs.python3} ${./server.py})
```

Comment:
Is the `()` subshell important here?

Reply:
I always use a subshell whenever I have a `cd`. I think it's not _required_ here, but I prefer globally changing the directory in a script

Reply:
I don't think the `cd` can leak out of the script anyway, but no worries.

It may be simpler to pass `"$doc_derivation"/share/doc` in to the `server.py` script rather than implicitly using the current working directory, though IDK and it's not really important.